### PR TITLE
Remove sass globbing and add targetExtension

### PR DESCRIPTION
**Pull request by max-scopp**
Related with #149, if a sass-globber is needed, the user should define that for their project inside their respective config.

Another problem was that you couldn't _properly_ use `sass-brunch` in combination with `postcss-brunch`, by adding the `targetExtension`  property, the resulting css will be routed through the proper plugins.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const promiseSpawnAndPipe = (cmd, args, env, data) => {
 };
 
 class SassCompiler {
-  constructor(cfg) {
+  constructor (cfg) {
```
Comment (denysdovhan) [inline index.js]: ☹️

File: index.js [hunk 2]
```diff
@@ -139,45 +141,46 @@ class SassCompiler {
...
-      },
-      (error, result) => {
-        if (error) {
-          return reject(formatError(source.path, error));
-        }
-        const data = result.css.toString().replace('/*# sourceMappingURL=a.css.map */', '');
-        const map = JSON.parse(result.map.toString());
-        resolve({data, map});
-      });
+          file: source.path,
```
Comment (denysdovhan) [inline index.js]: 4 spaces?

File: index.js [hunk 3]
```diff
@@ -139,45 +141,46 @@ class SassCompiler {
...
+          sourceMapEmbed: !this.optimize && this.config.sourceMapEmbed,
+        },
+        (error, result) => {
+          if (error) {
+            return reject(formatError(source.path, error));
+          }
+          const data = result.css.toString().replace('/*# sourceMappingURL=a.css.map */', '');
+          const includedPaths = result.stat.includePaths;
+          const map = JSON.parse(result.map.toString());
+          resolve({data, map, includedPaths});
```
Comment (denysdovhan) [inline index.js]: without `includedPaths`

why do we need that?

File: index.js [hunk 4]
```diff
@@ -211,30 +214,28 @@ class SassCompiler {
...
 
-  get getDependencies() {
-    return progeny({
-      rootPath: this.rootPath,
-      altPaths: this.includePaths,
-      reverseArgs: true,
-      globDeps: true,
-    });
+  getDependencies (file) {
+    return this._nativeCompileSync(file).includedPaths;
```
Comment (denysdovhan) [inline index.js]: progeny is still in deps. what's wrong with it?

**Comment by Janiczek** [comment]
If there's problem with parts of this PR, could we at least push `SassCompiler.prototype.targetExtension = 'css';`? That will make the resulting CSS files minified :+1:

**Pull request closed without merging**